scripts/create_release_tag.js

```diff
@@ -0,0 +1,57 @@
+// This scripts is used to create a new release tag for the current release branch
+// when we need to submit a PR to PD for updating the tidb-dashboard.
+// After this tag is pushed, it will trigger the CI to build a new tidb-dashboard release version.
+// Then we can run the manual-create-pd-pr.yaml workflow in the GitHub to create a PR to PD.
+
+const { execSync } = require('child_process');
+
+function getGitBranch() {
+  // master, release-7.6
+  return execSync('git rev-parse --abbrev-ref HEAD').toString().trim();
+}
+
+function getGitShortSha() {
+  // eb69e4fd
+  return execSync('git rev-parse --short HEAD').toString().trim();
+}
+
+function getGitLatestTag() {
+  // v7.6.0-alpha, v7.6.0-<sha>, v7.6.0, v7.6.1-<sha>, v7.6.1
+  return execSync('git describe --tags --dirty --always').toString().trim();
+}
+
+function createReleaseTag() {
+  const branch = getGitBranch();
+
+  if (!branch.match(/release-\d.\d$/)) {
+    console.error('Err: this is not a valid release branch');
+    return
+  }
+
+  const branchVer = branch.replace('release-', '');
+  const latestTag = getGitLatestTag();
+
+  if (!latestTag.startsWith(`v${branchVer}.`)) {
+    console.error(`Err: latest tag ${latestTag} doesn't match the branch ${branch}, you need to add the new tag manually`);
+    return
```

Comment:
It's better throw error to make it exits with a no-zero code.

Reply:
This script only runs in local, so it won't break CI flows.